bin/Powheg/run_pwg.py

```diff
@@ -1148,16 +1133,53 @@ def runhnnlo(folderName, njobs, QUEUE):
             os.system('cp -p '+args.inputTemplate+' '+args.folderName+'/powheg.input')
 
         os.system('rm -rf JHUGen.input')
-        inputJHUGen = '/'.join(powInputName.split('/')[0:-1])+'/JHUGen.input'
+        inputJHUGen = args.inputJHUGen
+        if args.inputJHUGen == "":
+            inputJHUGen = '/'.join(powInputName.split('/')[0:-1])+'/JHUGen.input'
+
         if not os.path.exists(inputJHUGen) :
-            os.system('wget --quiet --no-check-certificate -N http://cms-project-generators.web.cern.ch/cms-project-generators/'+inputJHUGen+' -O '+args.folderName+'/JHUGen.input')
+            m_ret = os.system('wget --quiet --no-check-certificate -N http://cms-project-generators.web.cern.ch/cms-project-generators/'+inputJHUGen+' -O '+args.folderName+'/JHUGen.input')
+            if ((m_ret>>8) & 255) != 0 :
+                os.system('rm -rf '+args.folderName+'/JHUGen.input')
+#            print 'return value: ', ((m_ret>>8) & 255), m_ret
+
 #            if os.path.exists('JHUGen.input') :
 #                os.system('cp -p JHUGen.input '+args.folderName+'/.')
 #            if os.path.exists(args.inputJHUGen) :
 #                os.system('cp -p '+args.inputJHUGen+' '+args.folderName+'/JHUGen.input')
         else :
             os.system('cp -p '+inputJHUGen+' '+args.folderName+'/JHUGen.input')
 
+        if os.path.exists(args.folderName+'/powheg.input') :
+            #card_in = ConfigParser.ConfigParser()
+            test_pdf1 = 0
+            test_pdf2 = 0
+
+            default_pdf = "306000"  # for 5 flavours
+
+            if args.prcName=="ST_tch_4f" or args.prcName=="bbH" or args.prcName=="Wbb_dec" or args.prcName=="Wbbj" :
+                default_pdf = "320900"  # for 4 flavours
+
+            for line in open(args.folderName+'/powheg.input') :
+                if 'lhans1' in line and len(line.split(' ')) >= 2:
+                    test_pdf1 = line.split(' ')[1].strip()
```

Comment:
these conditions 
```
test_pdf1 = line.split(' ')[1].strip()
test_pdf2 = line.split(' ')[1].strip()
```

are not sufficient. if the line is like
```
lhans1   306000
```
i.e. contains multiple spaces, it will not work!
I will merge then push a fix (i.e. simply use `split()` )

Reply:
Ah, ok. I should use [-1] instead. The strip() is simply to get rid of possible new line character at the tail.

Reply:
Ya, surely just use split() will prevent the issue.